sources/packages/web/src/components/common/InstitutionUserSummary.vue

```diff
@@ -213,29 +213,17 @@ export default {
       sortOrder = DataTableSortOrder.ASC,
     ) => {
       loading.value = true;
-      switch (clientType.value) {
-        case ClientIdType.Institution:
-          usersListAndCount.value =
-            await InstitutionService.shared.institutionSummary(
-              page,
-              pageCount,
-              searchBox.value,
-              sortField,
-              sortOrder,
-            );
-          break;
-        case ClientIdType.AEST:
-          usersListAndCount.value =
-            await InstitutionService.shared.institutionSummaryForAEST(
-              props.institutionId,
-              page,
-              pageCount,
-              searchBox.value,
-              sortField,
-              sortOrder,
-            );
-          break;
-      }
+      usersListAndCount.value =
+        await InstitutionService.shared.institutionUserSummary(
+          {
+            page: page,
```

Comment:
Why so much space here ?

Reply:
It is formatted by prettier. What is the issue here @henryoforeh-aot ?

![image](https://user-images.githubusercontent.com/54600590/164533601-1b5f7230-87ea-407f-a594-71e03f1c6193.png)